src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java

```diff
@@ -259,7 +259,7 @@ protected IAlignmentLimits getInitialAlignmentLimits() {
     protected GT_Multiblock_Tooltip_Builder createTooltip() {
         GT_Multiblock_Tooltip_Builder tt = new GT_Multiblock_Tooltip_Builder();
         tt.addMachineType("Powered Spawner")
-            .addInfo("Controller block for the Extreme Entity Crusher")
+            .addInfo("Controller block for the EEC : Extreme Entity Crusher")
```

Comment:
EEC should be written in a separated line like in mega apiary (`AKA. EEC`)

Reply:
btw shouldn't the aka be lowercase ?